src/com/hotmail/kalebmarc/textfighter/main/Weapon.java

```diff
@@ -213,26 +249,67 @@ public void dealDam() {
             }
         }
 
+        displayDamageDealt(damageDealt, didCritical);
+
+    }
+
+    // Standard setter for criticalChance
+    // Can be used for future enhancements like weapon upgrades
+    public void setCriticalChance(double chance){
+        // Set a default to not go under 0
+        if(chance < 0)
+        {
+            this.criticalChance = .01;
+        }else{
+            this.criticalChance = chance;
+        }
+    }
+
+    // Standard getter for criticalChance
+    public double getCriticalChance(){
+        return this.criticalChance;
+    }
+
+    // Displays the damage dealt to console
+    public void displayDamageDealt(int damageDealt, boolean didCritical){
         //Display stuff
         com.hotmail.kalebmarc.textfighter.player.Stats.totalDamageDealt += damageDealt;
         com.hotmail.kalebmarc.textfighter.player.Xp.setBattleXp(damageDealt, true);
         if(!Enemy.get().takeDamage(damageDealt)) { // !dead
-	        Ui.cls();
-	        Ui.println("----------------------------------------------------");
-	        Ui.println("You have attacked a " + Enemy.get().getName() + "!");
-	        Ui.println("You dealt " + damageDealt + " damage with a " + this.name);
-	        Ui.println("----------------------------------------------------");
-	        Ui.println("Your health: " + com.hotmail.kalebmarc.textfighter.player.Health.getStr());
-	        Ui.println("Enemy health: " + Enemy.get().getHeathStr());
-	        Ui.println("----------------------------------------------------");
-	        Ui.pause();
-	
-	        if (Enemy.get().getHealth() <= Enemy.get().getHealthMax() / 3){
-	            Enemy.get().useFirstAidKit();
-	        }
-        } 
+            Ui.cls();
+            Ui.println("----------------------------------------------------");
+            Ui.println("You have attacked a " + Enemy.get().getName() + "!");
+
+            // Conditional to check whether critical hit or not to display correct message
+            if(didCritical)
+            {
+                Ui.print("Critical hit!");
+                Ui.println("You dealt " + damageDealt + " damage with a " + this.name);
+            }else{
+                Ui.println("You dealt " + damageDealt + " damage with a " + this.name);
+            }
+            Ui.println("----------------------------------------------------");
+            Ui.println("Your health: " + com.hotmail.kalebmarc.textfighter.player.Health.getStr());
+            Ui.println("Enemy health: " + Enemy.get().getHeathStr());
+            Ui.println("----------------------------------------------------");
+            Ui.pause();
+
+            if (Enemy.get().getHealth() <= Enemy.get().getHealthMax() / 3){

```

Comment:
```suggestion
            if (Enemy.get().getHealth() <= Enemy.get().getHealthMax() / 3) {

```

sorry, all these spacing suggestions are just me being picky 😄